### PR TITLE
Add preflight check

### PR DESCRIPTION
**Pull request by JakeNeyer**

### CODE REVIEW

File: terraform/modules/ecs/vars.tf [hunk 1]
```diff
@@ -41,6 +46,16 @@ variable "polytomic_root_user" {
   default     = ""
 }
 
+variable "polytomic_data_path" {
```
Comment (nyergler) [inline terraform/modules/ecs/vars.tf:49]: Does this also need to be referenced by the mount in the task definition?

File: terraform/modules/ecs/main.tf [hunk 2]
```diff
@@ -13,6 +13,7 @@ locals {
     polytomic_port = var.polytomic_port,
     env = {
       ROOT_USER                        = var.polytomic_root_user,
+      LOCAL_DATA_PATH                  = var.polytomic_data_path,
```
Comment (nyergler) [inline terraform/modules/ecs/main.tf:16]: I guess `LOCAL_DATA=true` is set in the environment elsewhere? I guess if the preflight worked it _must_ be :).

**Review by nyergler — APPROVED** [review]
One question re: carrying the `LOCAL_DATA_PATH` value through to the task definition, otherwise looks sane. Thanks!

File: terraform/modules/ecs/main.tf (hunk 2 above)
Comment (JakeNeyer) [reply terraform/modules/ecs/main.tf:16]: Oh interesting. I set this in order to get the preflight log to output correctly. I suppose we also need to add the `LOCAL_DATA` boolean if we want syncs to actually use it

**Pull request merged**